### PR TITLE
fix(ErrorSummary,SectionalBanner): fix spacing and border-radius

### PR DESCRIPTION
**Pull request by carlafranca**
DS-726 
Fix All Spacing and Border as per ticket.
Also included  DS-725 changes as they were related.

Please use figma design and not the spacing added to the description.

### CODE REVIEW

File: packages/react/src/components/error-summary/error-summary.tsx [hunk 1]
```diff
@@ -74,7 +78,7 @@ export const ErrorSummary: VoidFunctionComponent<ErrorSummaryProps> = ({
     ));
 
     return (
-        <SectionalBanner
+        <StyleSectionalBanner
```
Comment (LarryMatte) [inline packages/react/src/components/error-summary/error-summary.tsx:81]: I think we are using "Styled" with a "d" in the other components. so _StyledSectionalBanner_

**Review by LarryMatte — CHANGES_REQUESTED** [review]
A few comments here and there, and it appears that everything mentioned in the story has been addressed. However, for some unknown reason, there are still a few disparities in comparison to the visual design.

@maboilard, could you please double-check the requirements to ensure everything aligns as expected?

Screenshot 2023-11-02 at 11 51 01 AM

File: packages/react/src/components/sectional-banner/sectional-banner.tsx [hunk 2]
```diff
@@ -61,9 +61,10 @@ function abstractContainer(
     return styled.section<AbstractContainerProps>`
         background-color: ${bgColor};
         border: 1px solid ${(props) => (color ? props.theme.notifications[color] : props.theme.main['primary-3'])};
+        border-radius: var(--border-radius-2x);
         box-sizing: border-box;
         line-height: ${({ isMobile }) => getLineHeight(isMobile)}px;
-        padding: ${(props) => (props.isMobile ? 'var(--spacing-3x) var(--spacing-2x)' : 'var(--spacing-2x)')};
+        padding: ${(props) => (props.isMobile ? 'var(--spacing-3x) var(--spacing-2x)' : 'var(--spacing-2x) var(--spacing-3x)')};
```
Comment (pylafleur) [inline packages/react/src/components/sectional-banner/sectional-banner.tsx:67]: The story prescribed a padding-bottom of 20px. Given the 4px space above the title and icon, that would make the spacing even between the top and the bottom.

File: packages/react/src/components/error-summary/error-summary.tsx
Comment (pylafleur) [inline packages/react/src/components/error-summary/error-summary.tsx:1]: The margin around the ErrorList `ul` has not been modified

`margin: 8px 0 8px 8px` (using spacing variables) as described in DS-726

File: packages/react/src/components/error-summary/error-summary.tsx [hunk 3]
```diff
@@ -22,6 +22,10 @@ const ErrorItem = styled.li`
     margin-bottom: var(--spacing-1x);
 `;
 
+const StyledSectionalBanner = styled(SectionalBanner)`
+    padding-bottom: var(--spacing-3x);
```
Comment (pylafleur) [inline packages/react/src/components/error-summary/error-summary.tsx:26]: This is not necessary. With adjustments on the `ul`, the space at the bottom matches the design.

**Review by pylafleur — CHANGES_REQUESTED** [review]

File: packages/react/src/components/sectional-banner/sectional-banner.tsx
[the remark this answers is not in the document]
Comment (carlafranca) [reply packages/react/src/components/sectional-banner/sectional-banner.tsx:1]: done

File: packages/react/src/components/error-summary/error-summary.tsx
Comment (carlafranca) [reply packages/react/src/components/error-summary/error-summary.tsx:1]: done

File: packages/react/src/components/error-summary/error-summary.tsx (hunk 3 above)
Comment (carlafranca) [reply packages/react/src/components/error-summary/error-summary.tsx]: done

File: packages/react/src/components/sectional-banner/sectional-banner.tsx (hunk 2 above)
Comment (carlafranca) [reply packages/react/src/components/sectional-banner/sectional-banner.tsx:67]: done, all styles aren't based on the story anymore but on the figma design

**Review by LarryMatte — APPROVED** [review]

**Review by pylafleur — APPROVED** [review]

**Pull request merged**